libs/pxweb2-ui/src/lib/components/Select/Select.tsx

```diff
@@ -220,7 +225,17 @@ function VariableBoxSelect({
           isOpen={isModalOpen}
           onClose={handleCloseModal}
         >
-          <div className={cl(classes.modalRadioList)}>
+          
+                <Radio 
+                  value={clickedItem?.value ?? ''}
+                  name='option'
+                  options={options}                  
+                  selectedOption={clickedItem?.value}
+                  onChange={handleRadioChange}
+                >                
+                </Radio>
+
+         {/* <div className={cl(classes.modalRadioList)}>
```

Comment:
Should this commented out code be removed?

Reply:
Removed commented code